easybuild/easyconfigs/p/PyTorch/USER.md

```diff
@@ -163,8 +272,6 @@ srun --cpu-bind=mask_cpu:$MYMASKS \
     the search path for executables, and you need to modify the job script to use
     `/runscripts/conda-python-distributed` instead.
 
-The `get-master` script is a Python script to determine the master node for communication.
-
 In the above example we do download all files, which of course is in general not a good idea, in particular
 if those files are only read anyway. However, the example itself further downloads data it needs.
```

Comment:
Could you check this block again that it makes sense? Maybe I'm just tired but there's something in the meaning of these sentences that I don't get. (We download the .py and .dat files , should we not download them maybe or what does this mean?)

Reply:
Changed the script so that the download in the script is not longer needed.